### ISSUE TITLE
Incompatible libcurl version. 

### ISSUE THREAD

**Issue by steventheuniverse** [JCMais/node-libcurl#115]
>>> import UnbinnedAnalysis
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/StevenZakirov/Desktop/Fermi/ScienceTools-v11r5p3-fssc-20180124-x86_64-apple-darwin16.7.0/x86_64-apple-darwin16.7.0/lib/python/UnbinnedAnalysis.py", line 12, in <module>
    import pyLikelihood as pyLike
  File "/Users/StevenZakirov/Desktop/Fermi/ScienceTools-v11r5p3-fssc-20180124-x86_64-apple-darwin16.7.0/x86_64-apple-darwin16.7.0/lib/python/pyLikelihood.py", line 7, in <module>
    import lib_pyLikelihood
ImportError: dlopen(/Users/StevenZakirov/Desktop/Fermi/ScienceTools-v11r5p3-fssc-20180124-x86_64-apple-darwin16.7.0/x86_64-apple-darwin16.7.0/lib/lib_pyLikelihood.so, 2): Library not loaded: /opt/local/lib/libcurl.4.dylib
  Referenced from: /Users/StevenZakirov/Desktop/Fermi/ScienceTools-v11r5p3-fssc-20180124-x86_64-apple-darwin16.7.0/x86_64-apple-darwin16.7.0/lib/libxerces-c-3.1.dylib
  Reason: Incompatible library version: libxerces-c-3.1.dylib requires version 9.0.0 or later, but libcurl.4.dylib provides version 7.0.0

- basically, I'm trying to install the software Enrico to run Fermi Science tools and it requires me to check if the python support of ST is installed and from what I can see the reason is libcurl.4.dylb doesn't provide the right version of the support libxerces-c-3.1.dylib needs.

**Comment by JCMais** [comment]
This is the repo for the Node.js bindings, not the Python one

**Comment by steventheuniverse** [comment]
thank you
